.github/workflows/ci.yml

```diff
@@ -122,8 +134,12 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install .  # install dependencies
+          pip install ".[all]"  # install dependencies
           pip install -r requirements-dev.txt  # install development dependencies and type stubs
+      - name: Install cleanvision
+        run: |
+          git clone https://github.com/cleanlab/cleanvision.git
+          pip install cleanvision/
       - name: Type check
         run: mypy --install-types --non-interactive cleanlab
   fmt:
```

Comment:
can you confirm none of your changes will affect the CI test we had before that checks cleanlab works without optional dependencies installed? thanks!

Reply:
Yes, it doesn't affect previous checks, https://github.com/cleanlab/cleanlab/blob/a0d72fcb36f1ce87d1a6022e491e4921c5f4267f/.github/workflows/ci.yml#L103